### ISSUE TITLE
Build for tvOS

### ISSUE THREAD

**Issue by siuying** [jxcore/jxcore#824]
Any chance we can have a tvos build?

**Comment by obastemur** [comment]
JXcore working on tvOS makes the biggest sense but I don't have an up to date device to take the responsibility. Did you try the iOS binary?

**Comment by siuying** [comment]
Yep, I tried. iOS binary will not work on Apple TV device as Apple requires build for tvos. 

I'm trying to build one myself, should finish git clone soon :)

**Comment by siuying** [comment]
I made couple changes in various build files, eventually stuck at this point:

``
:
In file included from In file included from ../deps/mozjs/incs/mozilla/double-conversion/utils.h../deps/mozjs/incs/mozilla/double-conversion/utils.h::3434:
:
In file included from In file included from ../deps/mozjs/incs/mozilla/Assertions.h../deps/mozjs/incs/mozilla/Assertions.h::1616:
:
In file included from In file included from ../deps/mozjs/incs/mozilla/Attributes.h../deps/mozjs/incs/mozilla/Attributes.h::1212:
:
In file included from In file included from ../deps/mozjs/incs/mozilla/Compiler.h../deps/mozjs/incs/mozilla/Compiler.h::1212:
:
../deps/mozjs/incs/mozilla/Char16.h../deps/mozjs/incs/mozilla/Char16.h::178178::1515::  errorerror: : unknownunknown  typetype  namename  'char16_t''char16_t'

typedef const char16_t* char16ptr_t;typedef const char16_t* char16ptr_t;

              ^              ^
``
